.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -4,52 +4,109 @@ title: "[Issue]: "
 labels: [bug, triage]
 body:
   - type: markdown
+    id: introduction
     attributes:
       value: |
-        Thanks for taking the time to report an issue. Before submitting a report, please do the following:
-        1. Please head to our forum or chat rooms and troubleshoot with volunteers if you haven't already. Links can be found here: https://jellyfin.org/contact/
-        2. Please search the bug tracker for similar issues. If you do find one, please comment there instead of opening a new bug report.
-        3. If you decide to open a new report, please provide as much detail as possible.
-        4. Please **ONLY** report **ONE** issue per report. If you are experiencing multiple issues, please open multiple reports.
+        ### Thank you for taking the time to report an issue!
+        Please keep in mind that Jellyfin is a [free and open-source](https://jellyfin.org/docs/general/about) project, made up entirely and exclusively of **volunteers** who donate their free time to the project.
+  - type: checkboxes
+    id: before-posting
+    attributes:
+      label: "This issue respects the following points:"
+      description: All conditions are **required**. Failure to comply with any of these conditions may cause your issue to be closed without comment.
+      options:
+        - label: This is a **bug**, not a question or a configuration issue. We recommend visiting our forum or chat rooms and troubleshoot with volunteers first. The links can be found [here](https://jellyfin.org/contact/).
+          required: true
+        - label: This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aissue+is%3Aopen+label%3Abug) _(I've searched it)_.
+          required: true
+        - label: I'm using an up to date version of Jellyfin Server stable, unstable or master. We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
+          required: true
+        - label: I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
+          required: true
```

Comment:
Why is the "only report one issue per report" item gone?

Reply:
Not sure. I wanted to include that. I might have forgotten that part after I moved things around.